src/lib/stitching/kaws/stitching.ts

```diff
@@ -118,9 +118,12 @@ export const kawsStitchingEnvironmentV2 = (
         "\n"
       )}): FilterArtworksConnection
     }
-    extend type HomePageMarketingCollectionsModule {
+    type HomePageMarketingCollectionsModule {
```

Comment:
You don't just have to extend existing types here, you can define wholly new ones! So for this type, since it just exists to be stitched (and doesn't need a resolver besides `{}`), it's a bit nicer to have it co-located with the stitching code IMO. That's the crux of the change basically. It also does allow you to get rid of the `unusedField` thing as well (a plus).